### DISCUSSION TITLE
Insomnia Dev Tools Network Tab not showing my messages

### DISCUSSION THREAD

**Discussion by terryclancy** [Kong/insomnia#8041] [Help]
In attempting to use Insomnia Dev Tools, when I send a successful https POST request to an API end point I do not see it in the “Network” tab of the Insomnia Dev Tools.  What I  DO see are several messages to sentry-ipc://sentry-electron.scope/sentry_key   and  https://api.insomnia.rest/graphql?projects  which appear to be some sort of logging messages (which I did not want or request BTW, but that is another issue) .  The question here is, why am I not seeing the message I sent and the related response ? See screen grab below taken a few seconds are pressing “Send”:

![InsomDevToolsIssue-1](https://github.com/user-attachments/assets/8e4aaa44-d483-4485-a92f-cbd816ecff8f)

Thanks all
Terry Clancy ClanceZ

**Comment by subnetmarco** [comment]
@filfreire maybe can help us shed some light as to why we don't see it.

**Reply by terryclancy** [reply]
Thanks @subnetmarco, but so far nothing back from @filfreire

**Reply by gatzjames** [reply]
Hi @terryclancy thanks for the feedback. 
We recently introduced this functionality of seeing the network requests related to the app to improve the development experience of the app.
The requests users send use a different network stack and are not visible in the dev tools currently. The reason is that the timeline and the general ui in the response pane should be more advanced to help you debug your APIs. Could you share more info about what information you are missing that we could provide in the app UI?
